src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -1,11 +1,25 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 public class Task6 {
+
+   public static int fact(int n) {
+        int result;
+
+        if(n==1) return 1;
+        result = fact(n-1) * n;
+        return result;
+    }
+
+
     public static double calculateS(double x) {
-        return 0d;
+
+        return  1 + x + Math.pow(x,2) / fact(2) + Math.pow(x,3) / fact(3) + Math.pow(x,4) / fact(4) ;
```

Comment:
It's OK, but what if you'll need to calculate 10 more terms ?
Try to solve this task using loop.